### PR TITLE
Update Node.js information for Ubuntu

### PR DESCRIPTION
**Pull request by mikaturunen**
Updating the actual Node.js setup information into a more standardized way of handling the Node.js environments and setups in general. 

Before anyone says it.. Running shell scripts from the Internet:
Running shell scripts from the Internet

### CODE REVIEW

File: doc/project/js-environments.md [hunk 1]
```diff
@@ -47,12 +47,22 @@ For more options of the PhantomJS environment, see
 
 ## <a name="node-on-ubuntu"></a> Node.js on Ubuntu
 
-On Ubuntu, the Node.js command from the [nodejs package](http://packages.ubuntu.com/utopic/nodejs) is called `nodejs` instead of `node` (when installed through the package manager). This will make the Node.js environment fail (since it simply calls `node`).
+Easiest way to handle Node.js versions and installations on Ubuntu (and in Linux systems in general) is to use [nvm](https://github.com/creationix/nvm). All instructions are included.
```
Comment (sjrd) [inline doc/project/js-environments.md]: "The easiest way"

File: doc/project/js-environments.md [hunk 2]
```diff
@@ -47,12 +47,22 @@ For more options of the PhantomJS environment, see
...
+
+Then run nvm to install wanted version of Node.js:
+    
+    nvm install 5.0
 
-1. Install [nodejs-legacy](http://packages.ubuntu.com/utopic/nodejs-legacy) which will add an alias called `node`
-2. Explicitly tell the Node.js environment the name of the command:
 
 {% highlight scala %}
 postLinkJSEnv := NodeJSEnv(executable = "nodejs").value
```
Comment (sjrd) [inline doc/project/js-environments.md]: Is this obsolete, then?

File: doc/project/js-environments.md [hunk 3]
```diff
@@ -47,12 +47,22 @@ For more options of the PhantomJS environment, see
...
-You have two options to solve this:
+In short it boils down to:
+
+    curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.29.0/install.sh | bash
+    . ~/.nvm/nvm.sh
+
+Add these lines to your ~/.bashrc, ~/.profile, or ~/.zshrc file to have it automatically sourced upon login:
+
+    export NVM_DIR="$HOME/.nvm"
+    [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" # This loads nvm
```
Comment (sjrd) [inline doc/project/js-environments.md]: I would not repeat these instructions here. They will become obsolete without us knowing. Let people click the link to the `nvm` readme.

File: doc/project/js-environments.md [hunk 4]
```diff
@@ -47,12 +47,22 @@ For more options of the PhantomJS environment, see
...
+
+Then run nvm to install wanted version of Node.js:
```
Comment (sjrd) [inline doc/project/js-environments.md]: "Then run `nvm` to install the version of Node.js that you want:"

**Comment by sjrd** [comment]
Thanks for your contribution. Could you please `amend` your commit with my comments, then force-push on your branch?

File: doc/project/js-environments.md (hunk 2 above)
Comment (mikaturunen) [inline doc/project/js-environments.md]: After installing node with nvm, nodejs is no longer available, it's just "node".

File: doc/project/js-environments.md (hunk 3 above)
Comment (mikaturunen) [inline doc/project/js-environments.md]: Good point. Removing.

**Comment by mikaturunen** [comment]
Thanks for the helpful feedback!

**Comment by sjrd** [comment]
LGTM

**Pull request merged**